styles/main.css

```diff
@@ -4,6 +4,9 @@ body {
 h1 {
   color: blue;
 }
+h2 {
+  color: blue;
+}
```

Comment:
Make sure to include rules for all the different heading types.  You have both h1 and h2 headings in your HTML files, so you need to colour them both